src/translations/dynamic/network-modifications-locale-en.js

```diff
@@ -0,0 +1,92 @@
+    'network_modifications.LINE_SPLIT_WITH_VOLTAGE_LEVEL':
+        'Splitting a line {computedLabel}',
+    'network_modifications.LINE_ATTACH_TO_VOLTAGE_LEVEL':
+        'Attaching line {computedLabel}',
+    'network_modifications.LINES_ATTACH_TO_SPLIT_LINES':
+        'Attaching lines to splitting lines {computedLabel}',
+    'network_modifications.LOAD_SCALING': 'Load scaling {computedLabel}',
+    'network_modifications.DELETE_VOLTAGE_LEVEL_ON_LINE':
+        'Deleting a voltage level on a line {computedLabel}',
+    'network_modifications.DELETE_ATTACHING_LINE':
+        'Deleting attaching line {computedLabel}',
+    'network_modifications.LOAD_CREATION': 'Creating load {computedLabel}',
+    'network_modifications.LOAD_MODIFICATION': 'Modifying load {computedLabel}',
+    'network_modifications.BATTERY_CREATION':
+        'Creating battery {computedLabel}',
+    'network_modifications.BATTERY_MODIFICATION':
+        'Modifying battery {computedLabel}',
+    'network_modifications.GENERATOR_CREATION':
+        'Creating generator {computedLabel}',
+    'network_modifications.GENERATOR_MODIFICATION':
+        'Modifying generator {computedLabel}',
+    'network_modifications.LINE_CREATION': 'Creating line {computedLabel}',
+    'network_modifications.LINE_MODIFICATION': 'Modifying line {computedLabel}',
+    'network_modifications.TWO_WINDINGS_TRANSFORMER_CREATION':
+        'Creating 2 windings transformer {computedLabel}',
+    'network_modifications.TWO_WINDINGS_TRANSFORMER_MODIFICATION':
+        'Modifying 2 windings transformer {computedLabel}',
+    'network_modifications.BRANCH_STATUS_MODIFICATION':
+        '{action, select, TRIP {Trip {computedLabel}} LOCKOUT {Lock out {computedLabel}} ENERGISE_END_ONE {Energise {computedLabel} on {energizedEnd}} ENERGISE_END_TWO {Energise {computedLabel} on {energizedEnd}} SWITCH_ON {Switch on {computedLabel}} other {Branch status modification {computedLabel}}}',
+    'network_modifications.SHUNT_COMPENSATOR_CREATION':
+        'Creating shunt compensator {computedLabel}',
+    'network_modifications.SHUNT_COMPENSATOR_MODIFICATION':
+        'Modifying shunt compensator {computedLabel}',
+    'network_modifications.GENERATOR_SCALING':
+        'Generator scaling {computedLabel}',
+    'network_modifications.VSC_CREATION': 'Creating vsc {computedLabel}',
+    'network_modifications.GROOVY_SCRIPT': 'Modification by script',
+    'network_modifications.EQUIPMENT_ATTRIBUTE_MODIFICATION':
+        '{equipmentAttributeName, select, open {{equipmentAttributeValue, select, true {Open {computedLabel}} other {Close {computedLabel}}}} other {Equipment modification {computedLabel}}}',
+    'network_modifications.creatingModification': 'Creating modification ...',
+    'network_modifications.deletingModification': 'Deleting modification ...',
+    'network_modifications.updatingModification': 'Updating modification ...',
+    'network_modifications.modifications': 'Updating modification list ...',
```

Comment:
the 4 above are not "dynamic"
(we can leave them, they have the same prefix)

Reply:
Yes, separating them is confusing. Maybe if we reorganize the way we name our keys, it would make sense and we could move them.